app/assets/javascripts/search_typeahead.js

```diff
@@ -0,0 +1,18 @@
+$(function () {
+    var suggester = new Bloodhound({
+        datumTokenizer: Bloodhound.tokenizers.obj.whitespace('value'),
+        queryTokenizer: Bloodhound.tokenizers.whitespace,
+        remote: {
+            url: '/search/autocomplete.json?query=%QUERY',
+            wildcard: '%QUERY'
+        }
+    });
+
+    $('#home_query, .header__search').typeahead({
+        minLength: 2,
+        highlight: true
+    }, {
+        name: "home_query",
+        source: suggester
+    });
+});
```

Comment:
Not sure if this is a problem with rubygems codestyle, but there is a missing newline at the end of the file